docs/reference/content/driver-async/index.md

```diff
@@ -9,7 +9,7 @@ title = "MongoDB Async Driver"
 
 ## MongoDB Async Java Driver Documentation
 
-The following guide provides information on using the callback-based MongoDB Async Java Driver 3.8.
+The following guide provides information on using the callback-based MongoDB Async Java Driver 3.9.
 
 {{% note %}}
 There are two higher level MongoDB Asynchronous Java Drivers available, that users may find easier to work with due to their friendlier APIs:
```

Comment:
Lets remove the RxJava from the list and mention that async is deprecated.

Reply:
Already done in last commit of the PR.